Remotion/Core/Core/Utilities/StringUtility.cs

```diff
@@ -448,7 +448,7 @@ private static object ParseEnumValue (Type underlyingType, string? value)
     {
       try
       {
-        return Enum.Parse (underlyingType, value, false);
+        return Enum.Parse (underlyingType, value!, false);
```

Comment:
What exactly should be the resolution of this bug, right now this would throw `ParseException(" is not a valid value for MyEnum.")`. Should `null` resulting in an empty string in the exception message be replaced with e.g. `<null>`, or is there a better resolution?

Reply:
It would remain a ParseException, it's just a dedicated check instead of brute forcing it. See Parse-method, line 381.

Reply:
From my understanding, the null check/exception should then happen in the Parse(...) method instead of the ParseEnum(...) method - am I wrong?

Reply:
Hmm...I just noticed, the ParseScalarValue method also happens to fall into the null-case. Probably needs to be handled as well, although the question is if value types and reference types require different handling.

So, I think it's probably better to handle the null-case inside the respectice methods.